src/main/java/edu/harvard/iq/dataverse/Dataset.java

```diff
@@ -43,6 +43,8 @@
  * @author skraffmiller
  */
 @NamedQueries({
+    @NamedQuery(name = "Dataset.findById", 
+                query = "SELECT o FROM Dataset o LEFT JOIN FETCH o.files WHERE o.id=:id"),
```

Comment:
Can you add a comment - this should only be used if you're going to use the files (otherwise use DatasetService.find() )? Would you ever want to use this outside the findDeep() method (at least people should consider that method rather than using this directly)?